lib/executor/instantiate/elem.cpp

```diff
@@ -100,8 +102,8 @@ Expect<void> Executor::initTable(Runtime::StoreManager &StoreMgr,
       ///   i32.const n
       ///   table.init idx
       ///   elem.drop idx
-    } else if (ElemSeg.getMode() ==
-               AST::ElementSegment::ElemMode::Declarative) {
+    } else if (ElemInst && ElemSeg.getMode() ==
```

Comment:
I think the `ElemInst` is checked at line 81. So we don't need an extra check here.

Reply:
I can remove it but the code analyzer will throw a warning here maybe.

Reply:
Please remove it. It is pretty sure that this is checked at line 81.

Reply:
Sure